recipes/r-downloader/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set version = '0.4' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-downloader
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: downloader_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/downloader_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/downloader/downloader_{{ version }}.tar.gz
+  sha256: 1890e75b028775154023f2135cafb3e3eed0fe908138ab4f7eff1fc1b47dafab
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: testthat
+requirements:
+  build:
+    - r-base
+    - r-digest
+
+  run:
+    - r-base
+    - r-digest
+
+test:
+  commands:
+    - $R -e "library('downloader')"  # [not win]
+    - "\"%R%\" -e \"library('downloader')\""  # [win]
+
+about:
+  home: https://github.com/wch/downloader
+  license: GPL-2
```

Comment:
GPL requires that the license be distributed with the package. Please include a `license_file` entry.